### PR TITLE
Update DDF schema to version 2

### PR DESCRIPTION
**Pull request by skateman**
https://github.com/ManageIQ/manageiq-ui-classic/issues/7277

### CODE REVIEW

File: app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb [hunk 1]
```diff
@@ -37,15 +37,15 @@ def params_for_create
             :name       => "uid_ems",
```
Comment (himdel) [inline app/models/manageiq/providers/ibm_cloud/power_virtual_servers/manager_mixin.rb:37]: no `:id` ?

**Comment by skateman** [comment]
@agrare can you please verify that the CI failure is unrelated?

**Comment by agrare** [comment]
Same issue as https://github.com/ManageIQ/manageiq-providers-nuage/pull/220#issuecomment-688845880, I'm not sure why cross-repo tests didn't pick this up unless these repos weren't included in the tests

**Comment by iv1111** [comment]
I've tested the changes. I don't know if I'm missing something but I get this:

image

I've pulled the changes from the upstream base repo MIQ and did bin/update. Then tried adding our provider via UI.

**Comment by agrare** [comment]
@iv1111 I believe there are some core UI dependencies that have to be merged first before this will work

**Comment by skateman** [comment]
@iv1111 you need https://github.com/ManageIQ/manageiq-ui-classic/issues/7277 to test this

**Comment by iv1111** [comment]
ok, i see now, thanks

**Comment by skateman** [comment]
> @iv1111 you need [ManageIQ/manageiq-ui-classic#7277](https://github.com/ManageIQ/manageiq-ui-classic/issues/7277) to test this

my bad, it's https://github.com/ManageIQ/manageiq-ui-classic/pull/7279

**Review by himdel — APPROVED** [review]

**Comment by iv1111** [comment]
I was recently working on the manageiq-ui-classic, so I had a path to my local repo instead of the official gem in the Gemfile. I had to update my local repo.

I've tested it now and see no errors when adding a new IBM cloud provider. The same Travis problem occured on my own PR now.

**Comment by skateman** [comment]
@miq-bot assign @Fryguy

**Pull request merged**